ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -19,7 +19,8 @@
 
 import { zodResolver } from '@hookform/resolvers/zod';
 import { createFileRoute, useNavigate } from '@tanstack/react-router';
-import { useForm } from 'react-hook-form';
+import { PlusIcon, TrashIcon } from 'lucide-react';
```

Comment:
Commit message nits:
- If referring to ORT, do so as "ORT", not "ort". However, we could also just say e.g. "run creation form", without "ORT".
- Avoid work duplication in "Add the ability of adding", maybe just "Allow to specify parameters..."

Reply:
Just curious about the validation: As I guess every input can be accepted as a string, what kind of input would fail? The only case I can think of right now where it should fail is an empty key. I guess we should allow empty values, though, as for some use-cases just the presence of a key is enough.